proto/sf/ethereum/type/v2/type.proto

```diff
@@ -189,10 +208,14 @@ message TransactionTrace {
   bytes s = 9;
 
   // GasUsed is the total amount of gas unit used for the whole execution of the transaction.
+  //
+  // Only available in DetailLevel: EXTENDED
   uint64 gas_used = 10;
 
   // Type represents the Ethereum transaction type, available only since EIP-2718 & EIP-2930 activation which happened on Berlin fork.
   // The value is always set even for transaction before Berlin fork because those before the fork are still legacy transactions.
+  //
+  // Only available in DetailLevel: EXTENDED
   Type type = 12;
```

Comment:
Say with Polygon, when there's a different endpoint called to get special transcations, we can't reuse this at all?